script/distanceOfErrMsg.pl

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/perl
+
+use Text::Levenshtein qw(distance);
+use Text::JaroWinkler qw(strcmp95);
+use Params::Validate qw/validate_pos/;

```

Comment:
It would be great to run tidy on the code as the quoting style is already inconsistent at this point. (see https://github.com/os-autoinst/openQA/blob/master/docs/Contributing.asciidoc#rules-for-commits=)